modules/exploits/linux/http/pulse_secure_gzip_rce.py

```diff
@@ -61,6 +61,11 @@
         {'type': 'url', 'ref': 'https://kb.pulsesecure.net/articles/Pulse_Security_Advisories/SA44601'},
         {'type': 'cve', 'ref': '2020-8260'}
     ],
+    'notes': {
+      'Stability': ['CRASH_SAFE'],
+      'SideEffects': ['ARTIFACTS_ON_DISK', 'CONFIG_CHANGES', 'IOS_IN_LOGS'],
```

Comment:
`IOS_IN_LOGS`?

Reply:
as a side note there doesn't seem to be any checking/normalization/verification of these values, as is made obvious by this goof